pages/dkp/kommander/2.1/networking/domains/index.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: layout.pug
+navigationTitle: Domains
+title: Required Domains
+menuWeight: 30
+excerpt: This section describes the required domains for Kommander
+beta: false
+enterprise: false
+---
+
+You must have access to the following domains through the customers networking rules so that Kommander can download all required images:
```

Comment:
```suggestion
You must have access to the following domains through the customer networking rules so that Kommander can download all required images:
```